source/getting-started/tutorials/sensors/index.md

```diff
@@ -0,0 +1,283 @@
+# Tutorial: Sensor Configuration, Feedback, & Profiling
+
+- **Goal:** Learn how to use the AMDC Timing Manager to get feedback from sensors
+- **Complexity:** 4 / 5
+- **Estimated Time:** TODO
```

Comment:
Don't forget to follow the tutorial to get a time estimate. Remember it will probably take a new user longer to work through than it takes you.